charts/datadog/values.yaml

```diff
@@ -632,6 +632,15 @@ datadog:
     container_scrubbing:
       enabled: true
 
+    # datadog.orchestratorExplorer.customResourcesCollection -- Defines custom resources for the orchestrator explorer to collect
+
+    # customResourcesCollection is required for RBAC creation if a custom orchestrator explorer configuration is provided in `clusterAgent.confd` or `clusterAgent.advancedConfd`
+    # Each item should follow group/version/name, for example
+    # customResourcesCollection:
+    #   - datadoghq.com/v1alpha1/datadogmetrics
+    #   - datadoghq.com/v1alpha1/watermarkpodautoscalers
+    customResourcesCollection: []
```

Comment:
do you mind if we call it `customResources` instead? i find that a little more intuitive with the list type